src/js/bun/sqlite.ts

```diff
@@ -91,6 +92,13 @@ const constants = {
   SQLITE_FCNTL_RESET_CACHE: 42,
 };
 
+// https://www.sqlite.org/c3ref/c_alter_table.html
+const sqliteActionCodes = {
```

Comment:
We can use `const enum` and avoid paying the cost of the property access

```suggestion
const enum sqliteActionCodes {
```